image-builder/nsdax.c

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright(c) 2013-2019 Intel Corporation. All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of version 2 of the GNU General Public License as
```

Comment:
Just mentioning that the license for this file (GPL-2.0) is different to the license for the containing osbuilder repo project (Apache-2.0).

/cc @grahamwhaley.

Reply:
Good pick up. IANAL, but I believe as long as you are not linking this to any of our (apache2) code (and I think you are not), then you are probably good.
This file local license will I think over-rule the top level license. I don't know if we have to say something at the top level about 'some files may be under a different license' now though.

Overall, if you can confirm you are not linking this against any apache2 code, then I'm not going to worry about it.
Some attribution somewhere about where the code came from might be good if we did not add one already..

Reply:
I confirm, I'm not liking this against any apache2 code

Reply:
It would be interesting to discuss if we wanted to switch our LICENSE files to use the Debian machine readable copyright file format since that handles this sort of scenario nicely:

- https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/